plugins/cisco_umbrella_destinations/icon_cisco_umbrella_destinations/actions/dDelete/action.py

```diff
@@ -0,0 +1,35 @@
+import insightconnect_plugin_runtime
+from .schema import DDeleteInput, DDeleteOutput, Input, Output, Component
+from insightconnect_plugin_runtime.exceptions import PluginException
+
+# Custom imports below
+
+
+class DDelete(insightconnect_plugin_runtime.Action):
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="dDelete",
+            description=Component.DESCRIPTION,
+            input=DDeleteInput(),
+            output=DDeleteOutput(),
+        )
+
+    def run(self, params={}):
+        destination_list_id = params.get(Input.DESTINATIONLISTID)
+
+        # Convert input str 1234 5678 into format [1234, 5678]
+        # x = '1234 5678'
+        # x.split(" ") == ['1234', '5678']
+        payload = params.get(Input.PAYLOAD).split(" ")
+
+        # This handles any characters in the IDs which are not numerical values
+        for index, i in enumerate(payload):
```

Comment:
I'm currently working on the unit tests right now so no it is not.
This loop isn't exactly essential to the functionality of the code, and is only used for this action.
I can move it to the util folder if this is better

Reply:
I'm not sure if you necessarily need to move this loop out of the action and into utils, but, it would work fit nicely into a more business logic-oriented function within this action, just split out from the run function.

A nice separation is having API-specific/technical code in utils, and business logic for actions contained within the actions (but still separated from the run function).